committeeoversightapp/models.py

```diff
@@ -72,23 +140,33 @@ class StaticPage(Page):
 
 
 class LandingPage(Page):
-    body = StreamField([
-        ('heading', blocks.CharBlock(classname='full title', icon='openquote')),
-        ('paragraph', blocks.RichTextBlock()),
-        ('image', ImageChooserBlock()),
-        ('button',
-         blocks.StructBlock([
-            ('button_text', blocks.CharBlock()),
-            ('button_link', blocks.URLBlock())
-            ],
-         icon='site'))
-    ])
+    body = RichTextField()
 
     # Editor configuration
     content_panels = Page.content_panels + [
-        StreamFieldPanel('body'),
+        FieldPanel('body'),
     ]
 
+    def get_context(self, request):
+        context = super(LandingPage, self).get_context(request)
+        context['house_committees'] = CommitteeOrganization.objects.all().filter(
+                                    classification='committee'
+                                ).filter(
+                                    parent_id__name__in=['United States House of Representatives']
+                                )
+        context['senate_committees'] = CommitteeOrganization.objects.all().filter(
+                                    classification='committee',
+                                    parent_id__name__in=['United States Senate']
+                                )
+
+        committees = context['house_committees'] | context['senate_committees']
+        context['committees'] = committees.order_by('name')
+
+        print("helskdjlfksgj")
```

Comment:
`helskdjlfksgj`!!!!

Reply:
ugh you're learning all my advanced testing secrets